patches/Terraclient/Terraria/Terraclient/Commands/MystagogueCommand.cs

```diff
@@ -377,6 +377,14 @@ public class MystagogueCommand
 				})
 				.Build();
 
+			Create("refills", "Toggles the Favorited Item Refills Cheat in the Cheat Menu.")
+				.AddParameters(new List<CommandArgument>())
+				.AddAction(_ => {
+					CheatHandler.GetCheat<RefillsCheat>().Toggle();
+					CheatCommandUtils.Output(false, "Favorited item refills toggled " + (CheatHandler.GetCheat<ToolGodCheat>().isEnabled ? "on" : "off") + ".");
+				})
+				.Build();
+
```

Comment:
1. Localization (I'd rather you localize all other strings as well).
2. Why not add commands for every other cheat, too? It's be incredibly easy to just, like, use reflection to automatically add these cheats. Also, ternaries used for contextual strings are gross (you're also reffing ToolGodCheat, not RefillsCheat.